### PR TITLE
 Hackathon Submission: Search Ideas List

### PR DESCRIPTION
**Pull request by leonyangela**
# Hackathon Submission: Search Ideas List

## Name of Team Lead
- Leoni Angela

## GitHub handles of Team Members  
_Enter the GitHub handles of all your team members (including yourself) as separate bullet points_
- @leonyangela 

### Are all of your team members registered on our [Hackathon Signup page](https://apwr.dev/htf24-hackathon)?
_This is a mandatory step for all participants to be considered eligible for judging_

- [x] Yes
- [ ] No

---

## Rules and Code of Conduct  
By submitting this PR, you agree to follow our Rules and Code of Conduct.

- [x] Yes

---

## Anything Else?  
_Any other feedback, queries or information you would like to share with us?_

### CODE REVIEW

**Comment by adityaoberai** [comment]
@leonyangela please make a copy of the `template_submission` file instead of editing it, as specified in the repo readme. Also, please upload your project codebase into your own repository and link that in your submission, instead of adding the project codebase here

**Comment by leonyangela** [comment]
Hi @adityaoberai, I have updated the submission, thank you.

File: .gitignore
Comment (adityaoberai) [inline .gitignore:1]: Please remove this file from the PR

File: submissions/template_submission.md
Comment (adityaoberai) [inline submissions/template_submission.md:1]: Please remove this edit from the PR

**Review by adityaoberai — CHANGES_REQUESTED** [review]

File: submissions/template_submission.md
Comment (leonyangela) [reply submissions/template_submission.md:1]: fixed, please check it, thanks

**Review by adityaoberai — APPROVED** [review]

**Pull request merged**